Adyen/Core/Core Protocols/Component.swift

```diff
@@ -65,9 +65,9 @@ public extension Component {
 }
 
 private enum AssociatedKeys {
-    internal static var isDropIn = "isDropInObject"
+    internal static var isDropIn: Void?
```

Comment:
This is smart! Where did you learn this approach?

Reply:
https://github.com/atrick/swift-evolution/blob/diagnose-implicit-raw-bitwise/proposals/nnnn-implicit-raw-bitwise-conversion.md#workarounds-for-common-cases

Reply:
ok, swiftling getting smarter 🤔